pkgs/development/python-modules/azure-core/default.nix

```
@@ -15,16 +15,26 @@
 }:
 
 buildPythonPackage rec {
-  version = "1.23.1";
+  version = "1.24.0";
   pname = "azure-core";
   disabled = isPy27;
```

See https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/setup.py#L68

```suggestion
  disabled = pythonOlder "3.6";
```

---

Maybe in a separate PR?